apps/betterangels-backend/clients/types.py

```diff
@@ -104,10 +104,34 @@ def search(
 
         queryset = queryset.filter(reduce(and_, q_objects))
 
-        return (
-            queryset,
-            Q(),
-        )
+        return (queryset, Q())
+
+    @strawberry_django.filter_field
+    def full_name_search(
+        self,
+        queryset: QuerySet,
+        info: Info,
+        value: Optional[str],
+        prefix: str,
+    ) -> Tuple[QuerySet[ClientProfile], Q]:
+        """
+        Returns only exact match on first and last name (case insensitive).
+        Requires a "," delimiter to accommodate spaces in first/last names:
+        "{FIRST NAME},{LAST NAME}"
+
+        Ignores middle names and partial first name matches, e.g., "Marc,Ramirez" != "Marc Anthony,Ramirez"
```

Comment:
Could we pass in specific params to get the exact search needed? We may be relying a bit much on the comments for implementation detail, down to the order of search params and which fields are ignored. I'd prefer this to be specific.

Reply:
kk, i can make a query input like: 
```{"firstName": "Marc Anthony", "lastName": "Ramirez"}``` 
and middle name can be optional